src/CodeGenHelpers/EnumBuilder.cs

```diff
@@ -17,10 +17,25 @@ internal EnumBuilder(string name, CodeBuilder builder)
 
         public CodeBuilder Builder { get; }
 
+        public DocumentationComment XmlDoc { get; } = new DocumentationComment();
```

Comment:
this should probably be private and only exposed via the helper methods on the EnumBuilder.